pyocd/target/family/target_lpc5500.py

```diff
@@ -234,6 +237,13 @@ def unlock(self, dm_ap):
             return
         return
 
+
+    def reset(self, reset_type):
+        # unlock debug access after reset
+        super(LPC5500Family, self).reset(reset_type)
+
+        self.check_locked_state(None)
```

Comment:
Just noticed. We probably don't need this. Normally, `SoCTarget` will pass the reset request to its selected core, thus calling `CortexM_LPC5500.reset()`.

https://github.com/pyocd/pyOCD/blob/9b1a6319ef749a2c73d0cc5b341d60aea8c70047/pyocd/core/soc_target.py#L207-L214

Reply:
yes, it is not necessary. Removing it.